baml_language/forks/aws-config/systest/run-e2e.sh

```
@@ -0,0 +1,360 @@
+
+  EC2_INSTANCE_ID="$($AWS ec2 run-instances --image-id "$AMI" --instance-type t3.micro \
+    --iam-instance-profile Name="$EC2_IPROF" --subnet-id "$SUBNET" --associate-public-ip-address \
+    --security-group-ids "$SG" \
+    --tag-specifications "ResourceType=instance,Tags=[{Key=Name,Value=baml-systest-$SUFFIX}]" \
+    --query 'Instances[0].InstanceId' --output text)"
+  echo "  instance=$EC2_INSTANCE_ID; waiting for running + SSM…"
+  $AWS ec2 wait instance-running --instance-ids "$EC2_INSTANCE_ID"
+
+  SSM_ONLINE=""
+  for _ in $(seq 1 20); do
+    if [ "$($AWS ssm describe-instance-information --filters Key=InstanceIds,Values=$EC2_INSTANCE_ID --query 'InstanceInformationList[0].PingStatus' --output text 2>/dev/null)" = "Online" ]; then SSM_ONLINE=1; break; fi
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)